sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HiveClientVersions.scala

```diff
@@ -19,16 +19,12 @@ package org.apache.spark.sql.hive.client
 
 import scala.collection.immutable.IndexedSeq
 
-import org.apache.commons.lang3.{JavaVersion, SystemUtils}
 
 private[client] trait HiveClientVersions {
   private val testVersions = sys.env.get("SPARK_TEST_HIVE_CLIENT_VERSIONS")
   protected val versions = if (testVersions.nonEmpty) {
     testVersions.get.split(",").map(_.trim).filter(_.nonEmpty).toIndexedSeq
-  } else if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)) {
-    IndexedSeq("2.0", "2.1", "2.2", "2.3", "3.0", "3.1")
   } else {
-    IndexedSeq("0.12", "0.13", "0.14", "1.0", "1.1", "1.2", "2.0", "2.1", "2.2", "2.3", "3.0",
-      "3.1")
+    IndexedSeq("2.0", "2.1", "2.2", "2.3", "3.0", "3.1")
```

Comment:
https://github.com/apache/spark/blob/5d422155f1dae09f1631375d09e2f3c8dffba9a5/sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HivePartitionFilteringSuites.scala#L27

Is this `filterNot` no longer needed as well?

Reply:
You're awesome 👍